### ISSUE TITLE
escodegen library related to this project

### ISSUE THREAD

**Issue by dusknguyen** [MichaelXF/js-confuser#127]
escodegen library related to this project 

```
node_modules/escodegen/escodegen.js:2516
        result = this[type](expr, precedence, flags);
                           ^

TypeError: this[type] is not a function
    at CodeGenerator.generateExpression (/node_modules/escodegen/escodegen.js:2516:28)
```
Version in my package-lock.json: 2.3.0

**Config**

Config:

```js
{
  target: "node",
  preset: "low",
  renameGlobals: false
}
```

**Comment by shellhazard** [comment]
For anyone else running into this, there's a maintained fork of escodegen which doesn't have this issue: https://www.npmjs.com/package/escodegen-wallaby

Workaround:
- Fork this repository [(example)](https://github.com/shellhazard/js-confuser).
- `bun install escodegen-wallaby && bun remove escodegen` then find and replace `require("escodegen")` with `require("escodegen-wallaby)"`.
- Remove `dist` from `.gitignore`.
- `bun run build` then commit and push everything.
- Add to your project with `bun install $gh_username/js-confuser`.

**Comment by MichaelXF** [comment]
Fixed in 2.0 (Currently [2.0.0-alpha.2](https://www.npmjs.com/package/js-confuser/v/2.0.0-alpha.2)). The 2.0 version nows uses Babel's parser and generator and is more suited for modern JavaScript syntax.
